avm/ptn/authorization/policy-exemption/main.bicep

```diff
@@ -0,0 +1,150 @@
+metadata name = 'Policy Exemptions (All scopes)'
+metadata description = 'This module deploys a Policy Exemption at a Management Group, Subscription or Resource Group scope.'
```

Comment:
On a different note, has there been an AVM module proposal for this? I have not found the module in the MAR file which is a prerequisite to successfully publish.

cc: @matebarabas

Reply:
Mate had me create 3 separate ones: 

-https://github.com/Azure/Azure-Verified-Modules/issues/1405?notification_referrer_id=NT_kwDOAddDN7QxMjQzNDk1NzkwMzozMDg4NDY2Mw#issuecomment-2422706165
-https://github.com/Azure/Azure-Verified-Modules/issues/1420?notification_referrer_id=NT_kwDOAddDN7QxMjQ5MjIxNzYwMjozMDg4NDY2Mw#issuecomment-2379474840
-https://github.com/Azure/Azure-Verified-Modules/issues/1421?notification_referrer_id=NT_kwDOAddDN7QxMjQ5MjIyNjAyMTozMDg4NDY2Mw#issuecomment-2379473633

Reply:
@matebarabas we need to have a word 😄 Will ping you internally. @oZakari no worries, your module can stay as is, but something in the back needs updating.

Reply:
Hey @matebarabas, has there bin an update already? :)

Reply:
This has been resolved.